team_production_system/serializers/availability.py

```diff
@@ -79,4 +85,32 @@ def validate_end_time(self, value):
             )
         return value
 
-    # TODO: Add validation for start times
+
+# V2 API #
+# The mentor availability serializer
+class AvailabilitySerializerV2(serializers.ModelSerializer):
+
+    class Meta:
+        model = Availability
+        fields = ['pk', 'mentor', 'start_time', 'end_time', 'status']
+        read_only_fields = ('pk',)
+
+    def validate(self, data):
```

Comment:
How is this being called? I'm not calling for a fix, this is just a gap in my Django knowledge.

Reply:
Good question! DRF has a validate built-in method that picks this up. I am overriding part of it here.